db/appsock_handler.c

```diff
@@ -188,6 +190,21 @@ static void *thd_appsock_int(appsock_work_args_t *w, int *keepsocket,
     arg.table_name = strdup(COMDB2_STATIC_TABLE);
     arg.conv_flags = 0;
 
+    /* If admin flag is on that means we failed to dispatch in appsockpool, so we had to read socket inline. */
```

Comment:
Confused a bit by the comment.  Did you mean the header_read flag?

Reply:
Yes, initial change was to use admin flag only.. later I added header_read flag.. the code explains the logic clearly now.. will remove the comment.